codersdk/deployment.go

```diff
@@ -1799,6 +1812,37 @@ Write out the current server config as YAML to stdout.`,
 			Group:       &deploymentGroupClient,
 			YAML:        "webTerminalRenderer",
 		},
+		// Healthcheck Options
+		{
+			Name:        "Healthcheck Timeout",
```

Comment:
```suggestion
			Name:        "Health Check Timeout",
```

I think it's ok in code, but as a compound word, healthcheck is uncommon. (Same for all other human readable texts.)

Reply:
Thoughts on `HEALTHCHECK` versus `HEALTH_CHECK`?